### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by PHAGUN-JAIN**
updating readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
+# Pettamo
 
+**Live Demo**: <https://pettamo.netlify.app/>
+</n>
+**Video Pitch**: <https://www.youtube.com/watch?v=fvX_XDb5HnU>
 
 ## Objectives
+
 * Provide a fullstack development example using Astra DB as the storage backend
```
Comment (stellasis) [inline README.md]: remove this. we dont need objectives.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
 
-JAMstack + Netlify + Cassandra + React + Bootstrap
+  JAMstack + Netlify + Cassandra + React + Bootstrap
```
Comment (stellasis) [inline README.md]: If you want to elaborate the stack, please put under `Overview` section

File: README.md (hunk 1 above)
Comment (PHAGUN-JAIN) [reply README.md]: yup removed

File: README.md [hunk 3]
```diff
@@ -1,12 +1,36 @@
+# Pettamo
 
+**Live Demo**: <https://pettamo.netlify.app/>
```
Comment (stellasis) [inline README.md]: Put the overview heading here

File: README.md [hunk 4]
```diff
@@ -1,12 +1,36 @@
...
+```
+
+### What and how does project leverage Astra DB
 
+### Quick steps about where to look for how you have used Astra DB for judges
 
-### Things to Note:
- - The contents of this repo are based on [Jake's port](https://github.com/tjake/todo-astra-react-serverless/) of the [TodoMVC code](https://github.com/tastejs/todomvc/tree/master/examples/react) originally written by [Pete Hunt](https://github.com/petehunt).
- - The project is modified from https://github.com/huksley/todo-react-ssr-serverless.
+### Things to Note (reference repos or credits of open-source)
```
Comment (stellasis) [inline README.md]:
```suggestion
### Credits
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,36 @@
...
+**Video Pitch**: <https://www.youtube.com/watch?v=fvX_XDb5HnU>
 
 ## Objectives
+
 * Provide a fullstack development example using Astra DB as the storage backend
 
-JAMstack + Netlify + Cassandra + React + Bootstrap
+  JAMstack + Netlify + Cassandra + React + Bootstrap
+
+## Getting Started
```
Comment (stellasis) [inline README.md]:
```suggestion
## Getting Started Locally
```

File: README.md [hunk 6]
```diff
@@ -1,12 +1,36 @@
+# Pettamo
```
Comment (stellasis) [inline README.md:2]: A web app for online vet consultations & verified pet care services to better care for your pets 🐶 🐱

File: README.md [hunk 7]
```diff
@@ -1,12 +1,55 @@
...
 
+## Overview
+
```
Comment (stellasis) [inline README.md:4]: This project is submitted for Build Modern Data App Hackathon. Link to live demo is [here](https://pettamo.netlify.app/)

File: README.md (hunk 2 above)
Comment (stellasis) [reply README.md]: i think just remove this

File: README.md [hunk 8]
```diff
@@ -1,12 +1,55 @@
...
 
-## Objectives
 * Provide a fullstack development example using Astra DB as the storage backend
 
-JAMstack + Netlify + Cassandra + React + Bootstrap
+  JAMstack + Netlify + Cassandra + React + Bootstrap
+
+**Live Demo**: <https://pettamo.netlify.app/>
+</n>
+**Video Pitch**: <https://www.youtube.com/watch?v=fvX_XDb5HnU>
```
Comment (stellasis) [inline README.md]: remove

File: README.md [hunk 9]
```diff
@@ -1,12 +1,55 @@
...
+| **src**/**Components**             | components used in the app.                                  |
+| **src**/**ContextStore**           | Global Data.                                                 |
+| **src**/**Modules**                | sub-components used in components.                           |
+| **src**/**Routes**                 | JS handling routes and guarding unauthorized req.            |
+| **src**/**Services**               | axios config and handling.                                   |
+| **src**/**Utils**                  | some utility functions.                                      |
+
+### What and how does project leverage Astra DB
 
+### Quick steps about where to look for how you have used Astra DB for judges
```
Comment (stellasis) [inline README.md]: are you planning to populate this?

Comment (stellasis) [reply README.md]: if not pls remove

**Pull request merged**